doc/man3/EVP_DigestInit.pod

```diff
@@ -105,13 +104,12 @@ made, but EVP_DigestInit_ex() can be called to initialize a new operation.
 
 Can be used to copy the message digest state from B<in> to B<out>. This is
 useful if large amounts of data are to be hashed which only differ in the last
-few bytes. B<out> must be initialized before calling this function.
+few bytes.
 
 =item EVP_DigestInit()
 
-Behaves in the same way as EVP_DigestInit_ex() except the passed context B<ctx>
-does not have to be initialized, and it always uses the default digest
-implementation.
+Behaves in the same way as EVP_DigestInit_ex() except it always uses the
+default digest implementation.
```

Comment:
(Is the default currently sha1 for compatibility, such that we should actively recommend the ```_ex``` version over this one?

Reply:
It is using the default implementation in the sense that it isn't using any engine provided one that might have been explicitly set.

Reply:
i.e. it is calling `EVP_MD_CTX_reset` on the `ctx` before calling `EVP_DigestInit_ex` with a NULL engine argument.

Reply:
If we want to change the wording of this, we'll want to also change the wording for `EVP_CipherInit` in _EVP_EncryptInit.pod_.

I'd suggest as a different PR.